cluster-autoscaler/scale_down.go

```diff
@@ -46,6 +47,13 @@ const (
 	ScaleDownNodeDeleted ScaleDownResult = iota
 )
 
+const (
+	// DrainTimeClusterAutoscaler is an annotation added to the nodes that CA marks as unschedulable.
+	DrainTimeClusterAutoscaler = "clusterautoscaler.beta.kubernetes.io/drain_time"
+	// MaxGracefulTerminationTime is max gracefull termination time used by CA.
+	MaxGracefulTerminationTime = time.Minute
```

Comment:
I will do in the next PR. The next PR will be about reducing the number of arguments passed to scale up/down and putting all the config related params into a single struct.